docker-compose.override.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  firefly-db:
+    volumes:
+      - firefly-dev-storage:/var/lib/mysql
+  firefly-app:
```

Comment:
Totally forgot about that part. I only did the splitting into base/dev/prod instead of mounting the dev files as discussed. Will fix.